dots_server/models/xorm_blocker_dao.go

```diff
@@ -463,7 +463,12 @@ func GetBlockerConfiguration(customerId int, targetType string) (blockerConfigur
 		return
 	}
 
+    if (blockerConfig.CustomerId != customerId) {
```

Comment:
Not quite sure if this is the best way as this is the first time I'm writing Go.